### PR TITLE
 SignUp Screen

### PR DESCRIPTION
**Pull request by leo-he-1**

### CODE REVIEW

**Comment by leo-he-1** [comment]
image

**Comment by leo-he-1** [comment]
image

**Comment by leo-he-1** [comment]
image

File: mobile/src/screens/signup/SignUp.view.tsx [hunk 1]
```diff
@@ -1,19 +1,56 @@
 import React, {FC} from 'react';
-import {View, Text, Button} from 'react-native';
-import {styles} from './SignUp.style';
+import {View, Text} from 'react-native';
+import {TextInput, Button} from '../../components';
 import {TSignUpScreenViewProps} from './SignUp.type';
+import {styles} from './SignUp.style';
+import {size, color} from '../../theme';
+import {Container} from '../../components';
```
Comment (lasharela) [inline mobile/src/screens/signup/SignUp.view.tsx:7]: you already have components here

File: mobile/src/screens/signup/SignUp.view.tsx [hunk 2]
```diff
@@ -1,19 +1,56 @@
...
+import BackgroundImage from '../../assets/background/background.svg';
+import {Footer} from '../../components';
```
Comment (lasharela) [inline mobile/src/screens/signup/SignUp.view.tsx:9]: one more

File: mobile/src/screens/signup/SignUp.view.tsx [hunk 3]
```diff
@@ -1,19 +1,56 @@
...
-        title={`Go to ${title}`}
-        onPress={press}
-      />
-    </View>
+
+    return (
+
+      <Container 
+      isLoading={isLoading}
+      backgroundImage={require('../../assets/background/auth_background.png')}
```
Comment (lasharela) [inline mobile/src/screens/signup/SignUp.view.tsx:21]: authBackground

**Review by lasharela — APPROVED** [review]

**Review by Dulaya — APPROVED** [review]

**Review by raynesmike — APPROVED** [review]
I approve this

**Comment by raynesmike** [comment]
dont forget to fix conflicts first for the yarn

**Review by Josetf2396 — APPROVED** [review]

**Pull request merged**